packages/framework/azure-client/README.md

```diff
@@ -83,6 +83,27 @@ const azureClient = new AzureClient(config);
 const { fluidContainer, containerServices } = await azureClient.createContainer(schema);
 ```
 
+### Detached Fluid Containers
+
+A detached container is helpful when the initial objects need to be seeded with default data before the application starts. Since the container is not yet attached to the client, no ops will be recorded and the first user to load the container will start with that initial snapshot.
+
+Creating a detached Fluid Container follows the same flow as the pre-attached container, minus an attach step that needs to be performed before loading the application.
+
+```javascript
+const schema = {
+    name: "my-container",
+    initialObjects: {
+        /* ... */
+    },
+    dynamicObjectTypes: [ /*...*/ ],
+}
+const tinyliciousClient = new TinyliciousClient();
+const { fluidContainer, containerServices } = await azureClient.createDetachedContainer(schema);
+
+// Perform `set` methods on the DDSes in fluidContainer.initialObjects
```

Comment:
Might be helpful to give an example `set` call here

Reply:
I could leave one in the comment, but this demo isn't really set up to take advantage of a detached container.

The other option would be to create a brand new demo, though that might be a ticket for another day